### PR TITLE
Create: new model/migration - order

### PR DESCRIPTION
**Pull request by MonietsSemen**

### CODE REVIEW

File: src/controllers/user.controller.ts [hunk 1]
```diff
@@ -19,7 +19,10 @@ class UserController {
 
   @SafeController
   static async show(_req: Request, res: LoadedUserResponse, _next: NextFunction) {
-    res.json({ user: res.locals.user });
+    // eslint-disable-next-line no-shadow
+    const { user } = res.locals;
+    const orders = await user.getOrders();
```
Comment (DenisRupp) [inline src/controllers/user.controller.ts]: try to use eager load instead of separate find orders

File: src/controllers/order.controller.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  @SafeController
+  static async create(req: Request, res: Response, _next: NextFunction) {
+    const orderData = req.body as CreationAttributes<Order>;
+    const { userId } = req.body;
+    const user = await User.findByPk(userId);
+
+    const orders = await user?.createOrder(orderData);
+
+    if (!user) throw res.status(NOT_FOUND).send('User not found');
```
Comment (DenisRupp) [inline src/controllers/order.controller.ts:40]: This condition should be before await user?.createOrder(orderData);

File: src/controllers/user.controller.ts (hunk 1 above)
Comment (MonietsSemen) [reply src/controllers/user.controller.ts]: Ready

File: src/controllers/order.controller.ts (hunk 2 above)
Comment (MonietsSemen) [reply src/controllers/order.controller.ts:40]: Ready

**Pull request merged**